src/main/java/exchange/core2/core/common/MatcherEventType.java

```diff
@@ -30,6 +30,9 @@
     // After cancel/reduce order - risk engine should unlock deposit accordingly
     REDUCE,
 
+    // cancel orders of suspended users
+    SUSPEND,
```

Comment:
I think you shoud issue N * REDUCE events for each cancelled order. Introducing new event type will make events handling much more difficult.

Reply:
Done.